code/datums/supplypacks.dm

```diff
@@ -647,6 +647,20 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "russian clothing crate"
 	group = "Clothing"
 	contraband = 1
+
+/datum/supply_packs/contacts
+	name = "Contact Lenses"
+	contains = list(/obj/item/clothing/glasses/contacts/polarized,
+					/obj/item/clothing/glasses/contacts/polarized,
```

Comment:
I know cargo *can* get shades by ordering secvend, building a vendor, hacking it, and buying the shades, but you rarely see that because it's a pain compared to just ordering a crate, unlike glubbs.

Not to mention these are much stealthier than shades, right?

Reply:
dusty crate has shades in it